src/main/java/edu/tamu/scholars/middleware/discovery/component/solr/SolrIndexer.java

```diff
@@ -119,4 +127,4 @@ private String name() {
         return type.getSimpleName();
     }
 
-}
+}
```

Comment:
```suggestion

}
```
Add missing trailing new line.
It seems weird to add it here using the code suggest.
I hope it works correctly if used.